### PR TITLE
azure-storage-azcopy: v10.3.2 -> v10.3.4

### PR DESCRIPTION
**Pull request by colemickens**
#### Motivation for this change

Just a small package bump.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/azcopy/default.nix [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
   pname = "azure-storage-azcopy";
-  version = "10.3.2";
+  version = "10.3.4";
 
   src = fetchFromGitHub {
     owner = "Azure";
     repo = "azure-storage-azcopy";
-    rev = "v${version}";
-    sha256 = "0n4yns81kwwx725smsgqg8hc693ygqlzrgkqdrhrfszkpm205479";
+    rev = version;
```
Comment (colemickens) [inline pkgs/development/tools/azcopy/default.nix]: https://github.com/Azure/azure-storage-azcopy/issues/868

**Review by jonringer — APPROVED** [review]
LGTM
shows usage
```
[4 built, 0.0 MiB DL]
https://github.com/NixOS/nixpkgs/pull/80409
1 package built:
azure-storage-azcopy

**Comment by jonringer** [comment]
@GrahamcOfBorg build azure-storage-azcopy

**Comment by colemickens** [comment]
rebased, re-`nix-review`-ed.

```
[cole@xeep:~/code/nixpkgs-azcopy]$ nix-review rev HEAD
$ git -c fetch.prune=false fetch --force https://github.com/NixOS/nixpkgs master:refs/nixpkgs-review/0
$ git worktree add /home/cole/.cache/nixpkgs-review/rev-272b96ce99c46d29e63086f8452fe67c3de209b3/nixpkgs 7aa084521b34efd71e9b17728f605011a1565a36
Preparing worktree (detached HEAD 7aa084521b3)
Updating files: 100% (21112/21112), done.
HEAD is now at 7aa084521b3 Merge pull request #81694 from r-ryantm/auto-update/liblouis
$ nix-env -f /home/cole/.cache/nixpkgs-review/rev-272b96ce99c46d29e63086f8452fe67c3de209b3/nixpkgs -qaP --xml --out-path --show-trace
$ git merge --no-commit 272b96ce99c46d29e63086f8452fe67c3de209b3
Automatic merge went well; stopped before committing as requested
$ nix-env -f /home/cole/.cache/nixpkgs-review/rev-272b96ce99c46d29e63086f8452fe67c3de209b3/nixpkgs -qaP --xml --out-path --show-trace --meta
derivation 'gnome-tour-0.0.1' has invalid meta attribute 'override'
derivation 'gnome-tour-0.0.1' has invalid meta attribute 'overrideDerivation'
1 package updated:
azure-storage-azcopy (10.3.2 → 10.3.4)

$ nix build --no-link --keep-going --option build-use-sandbox relaxed -f /home/cole/.cache/nixpkgs-review/rev-272b96ce99c46d29e63086f8452fe67c3de209b3/build.nix
[4 built, 13 copied (54.9 MiB), 8.4 MiB DL]
1 package built:
azure-storage-azcopy

$ nix-shell /home/cole/.cache/nixpkgs-review/rev-272b96ce99c46d29e63086f8452fe67c3de209b3/shell.nix

[nix-shell:~/.cache/nixpkgs-review/rev-272b96ce99c46d29e63086f8452fe67c3de209b3]$ 

```

**Review by jonringer — APPROVED** [review]
diff LGTM

```
[4 built, 0.0 MiB DL]
https://github.com/NixOS/nixpkgs/pull/80409
1 package built:
azure-storage-azcopy
```

**Pull request merged**